spec/support/ansible_shared/automation_manager/configuration_script.rb

```diff
@@ -6,9 +6,9 @@
   let(:connection)   { double(:connection, :api => api) }
   let(:job)          { AnsibleTowerClient::Job.new(connection.api, "id" => 1) }
   let(:job_template) { AnsibleTowerClient::JobTemplate.new(connection.api, "limit" => "", "id" => 1, "url" => "api/job_templates/1/", "name" => "template", "description" => "description", "extra_vars" => {:instance_ids => ['i-3434']}) }
-  let(:manager)      { FactoryGirl.create(:automation_manager_ansible_tower, :provider, :configuration_script) }
+  let(:manager)      { manager_with_configuration_scripts }
```

Comment:
Make the `shared_example` signature explicitly taking in this (and the other 2) provided object as parameters?  This would make the code much clearer where they are drop-ins.